dm/pkg/checker/privilege.go

```diff
@@ -75,7 +72,23 @@ func (pc *SourceDumpPrivilegeChecker) Check(ctx context.Context) *Result {
 		return result
 	}
 
-	verifyPrivileges(result, grants, dumpPrivileges)
+	dumpPrivileges := map[mysql.PrivilegeType]struct{}{
+		mysql.SelectPriv: {},
+	}
+
+	switch pc.consistency {
+	case "auto", "flush":
```

Comment:
we can use  exported vars from dumpling later
i opened a issue to enhance this https://github.com/pingcap/tidb/issues/31064